src/main/webapp/WEB-INF/themes/desktop/ftl/article/figure_carousel.ftl

```diff
@@ -34,6 +34,7 @@
 
               <@siteLink handlerName="figureImage" queryParameters=(versionPtr + {"size": "inline", "id": figure.doi}) ; src>
                 <img src="${src?html}"
+                     loading="lazy"
```

Comment:
Do any of these wombat images look like good candiates?
```
10:41 $ rg -g **/*.ftl \<img ../wombat
../wombat/src/main/webapp/WEB-INF/themes/root/app/approot.ftl
46:  <img src="data:image/jpg;base64,${imageCode}"/>

../wombat/src/main/webapp/WEB-INF/themes/root/ftl/browse/issuesBody.ftl
57:          <img

../wombat/src/main/webapp/WEB-INF/themes/root/ftl/article/metricsTemplates.ftl
25:        <a href="<%=  url %>"><img id="<%=  source_name %>ImageOnArticleMetricsTab" src="<%=  imgSrc %>"
34:        <img id="<%= source_name %>ImageOnArticleMetricsTab" src="<%= imgSrc %>" alt="<%= linkText %> <%= name %>"

../wombat/src/main/webapp/WEB-INF/themes/root/ftl/article/figureList.ftl
18:        <img class="figure-image" src="${src}"

../wombat/src/main/webapp/WEB-INF/themes/root/ftl/article/authorItem.ftl
66:        <img id="orcid-id-logo" src="<@siteLink path="/resource/img/orcid_16x16.png"/>" width="16" height="16" alt="ORCID logo"/>

../wombat/src/main/webapp/WEB-INF/themes/mobile/ftl/browse/volumesBody.ftl
38:            <img

../wombat/src/main/webapp/WEB-INF/themes/mobile/ftl/article/figure.ftl
28:<img class="figure-img"

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/browse/volumesBody.ftl
33:            <img
73:                      <img src="${issueImgURL}" alt="${issue.displayName} Journal Cover"/>

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/home/articleCard.ftl
28:                  <img class="grayscale" src="${href}"/>
34:              <img src="${href}"/>

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/common/loadingcycle.ftl
23:<img class="spinner" src="<@siteLink handlerName="staticResource" wildcardValues=["img/loading_small.gif"] />" class="loading" />

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/article/articleLightbox.ftl
54:      <img class="main-lightbox-image" src=""/>
58:          <img class="aside-figure" src="<@siteLink handlerName="figureImage" queryParameters=(versionPtr + {"size": "inline"})/>&id=<%= figure.getAttribute('data-doi') %>" />

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/article/metricsTemplates.ftl
25:        <a href="<%=  url %>"><img id="<%=  source_name %>ImageOnArticleMetricsTab" src="<%=  imgSrc %>"
34:        <img id="<%= source_name %>ImageOnArticleMetricsTab" src="<%= imgSrc %>" alt="<%= linkText %> <%= name %>"

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/article/figure_carousel.ftl
36:                <img src="${src?html}"

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/article/figure.ftl
55:      <img class="figure-img"

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/home/issue.ftl
26:    <img src="${src}" class="current-img" alt="Current Issue"/>

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/common/footer/footer.ftl
44:	 		 <img src="${src}" alt="${alt}" class="logo-footer"/>

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/article/aside/crossmark.ftl
27:<a data-target="crossmark"><img width="150" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_BW_horizontal.svg"></a>

../wombat/src/main/webapp/WEB-INF/themes/desktop/ftl/article/aside/shareOptions.ftl
14:<li><a href="https://www.reddit.com/submit?url=${urlParameter}" id="shareReddit" target="_blank" title="Submit to Reddit"><img src="<@siteLink path="/resource/img/icon.reddit.16.png"/>" width="16" height="16" alt="Reddit">Reddit</a></li>
18:<li><a href="https://www.facebook.com/share.php?u=${urlParameter}&t=${titleFix}" id="shareFacebook" target="_blank" title="Share on Facebook"><img src="<@siteLink path="/resource/img/icon.fb.16.png"/>" width="16" height="16" alt="Facebook">Facebook</a></li>
```

Reply:
Figures and equations are almost entirely guaranteed to be heavy and loaded outside of the view port when a user first lands on the article page. And because they are inline, loading them just pushes the remaining content down without any sort of reflow. It would be a bit tricky to make sure there isnt any image loading UI artifacts with a lot of those. Think of this as 20% effort getting 80% of the gains. Once we refocus on site optimization again, we can keep pushing this sort of fixes.